app/medInria/resources/authors.html

```diff
@@ -24,4 +19,20 @@
 Sergio.Medina@inria.fr <br/>
 Theodore.Papadopoulo@inria.fr <br/>
 Stephan.Schmitt@inria.fr <br/>
-
+<br/>
+<b>MUSIC team</b>
+<br/>
+Bruno Sore<br/>
+Florent.Collot@ihu-liryc.fr<br/>
+Florian.Vichot@inria.fr<br/>
+Hakim.Fadil@inria.fr<br/>
+Hubert Cochet<br/>
+Jatin Relan<br/>
+Jean-Michel Thomas<br/>
+Loic.Cadour@inria.fr<br/>
+Mathilde.Merle@ihu-liryc.fr<br/>
+Maxime.Sermesant@inria.fr<br/>
+Mehdi.Juhoor@inria.fr<br/>
+Michael.Buckingham@inria.fr<br/>
+Pauline.Migerditichan@inria.fr<br/>
+Pierre Jais<br/>
```

Comment:
Most of these addresses will become obsolete over time. I would simply keep the names.
Also, names should be in alphabetical order by last name, not first name.